packages/x-date-pickers/src/internals/components/PickersPopper.tsx

```diff
@@ -340,10 +342,11 @@ export function PickersPopper(inProps: PickerPopperProps) {
     onDismiss,
     open,
     role,
-    placement,
+    placement = 'bottom',
```

Comment:
I'm applying the default value here to make sure all the slots inside this file get a `popperPlacement` property.